cmd/server/main.go

```diff
@@ -33,15 +33,30 @@ const BindingGoogleKey = "googleKey"
 const BindingProjectNumber = "projectNumber"
 
 const UpsSecretName = "unified-push-server"
+const UpsURI = "uri"
 const GoogleKey = "googleKey"
 
 // This is required because importing core/v1/Secret leads to a double import and redefinition
 // of log_dir
 type BindingSecret struct {
-	metav1.TypeMeta              `json:",inline"`
-	metav1.ObjectMeta            `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
-	Data       map[string][]byte `json:"data,omitempty" protobuf:"bytes,2,rep,name=data"`
-	StringData map[string]string `json:"stringData,omitempty" protobuf:"bytes,4,rep,name=stringData"`
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+	Data              map[string][]byte `json:"data,omitempty" protobuf:"bytes,2,rep,name=data"`
+	StringData        map[string]string `json:"stringData,omitempty" protobuf:"bytes,4,rep,name=stringData"`
+}
+
+type UpsURLData struct {
```

Comment:
We could have a global instance of this struct to store the data (instead of retrieving it every time). Or is there a better way?